sormas-backend/src/main/java/de/symeda/sormas/backend/common/FollowUpTaskCreationException.java

```diff
@@ -0,0 +1,9 @@
+package de.symeda.sormas.backend.common;
+
+@SuppressWarnings("serial")
+public class FollowUpTaskCreationException extends Exception {
```

Comment:
I'd suggest to give this a more generic name (TaskCreationException) that we can then re-use in the future (since this is already not only used for follow-up tasks, but also for contact investigation tasks)